### PR TITLE
#8 emit event when back to top is triggered

### LINKED ISSUE #8: Feature request: emit event when scroll to top is clicked
I would like to propose a feature request to emit and event when scroll top top is triggered.  I can submit a pull request in a little bit with the implementation

### PR DESCRIPTION
**Pull request by ventralnet**
Closes #8

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,6 +43,11 @@ components: { BackToTop }
 | bottom        | String           | 40px            | bottom position of the component                                         |
 | right         | String           | 30px            | right position of the component                                          |
 
+## Events
+
+| Name                   | Description                                                              |
+|------------------------|--------------------------------------------------------------------------|
+| triggered              | Fired when back to top element is clicked                                |
```
Comment (caiofsouza) [inline README.md]: Hey @ventralnet,
Thanks for the issue and the PR ;D

This feature is amazing!!

About the event name, what you think if we use `scrolled` and the description `Fired when page's scroll end` ?

File: src/BackToTop.vue [hunk 2]
```diff
@@ -65,6 +65,7 @@ export default {
      * @return {void}
      */
     backToTop () {
+      this.$emit('triggered');
       window.smoothscroll()
```
Comment (caiofsouza) [inline src/BackToTop.vue]: I think that is better to emit event after the `window.smoothscroll()`

**Comment by ventralnet** [comment]
Sounds good to me!

**Comment by ventralnet** [comment]
Updated with the changes mentioned

**Pull request merged**